opencti-platform/opencti-graphql/src/domain/stixObservable.js

```diff
@@ -76,7 +76,7 @@ export const findAll = args => {
 export const stixObservablesNumber = args => ({
   count: countEntities('stix_observables', args),
   total: countEntities('stix_observables', dissoc('endDate', args))
-  /*count: getSingleValueNumber(
+  /* count: getSingleValueNumber(
```

Comment:
To remove

Reply:
Done, and remove all commented Grakn code.